core/src/transaction.rs

```diff
@@ -74,53 +67,146 @@ impl rlp::Decodable for UnverifiedTransaction {
     }
 }
 
-impl rlp::Encodable for UnverifiedTransaction {
+impl TempTransaction {
+    /// Append object with a signature into RLP stream
+    fn rlp_append_sealed_transaction(&self, s: &mut RlpStream) {
+        s.begin_list(6);
+        s.append(&self.unsigned.seq);
+        s.append(&self.unsigned.fee);
+        s.append(&self.unsigned.network_id);
+        s.append(&self.unsigned.action);
+        s.append(&self.sig);
+        s.append(&self.signer_public);
+    }
+}
+
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct Verified(TempTransaction);
```

Comment:
Let's rename `Verified` to `VerifiedTransaction` and `Unverified` to `UnverifiedTransaction`.

Reply:
I renamed it!